cg/utils/utils.py

```diff
@@ -20,3 +20,10 @@ def build_command_from_dict(options: dict, exclude_true: bool = False) -> List[s
                 formatted_options.append(str(option_name))
                 formatted_options.append(str(option_value))
     return formatted_options
+
+
+def get_hamming_distance(str_1: str, str_2: str) -> int:
```

Comment:
I think it would be better if the parameters were called `sequence_a` and `sequence_b` or something, since we are in the domain of DNA sequencing and it is more descriptive thank `str_1`

Reply:
I wrote the function in `utils.py` because I wanted this to be general for any use, not only sequences. The specific functions  for comparing indexes are in `cg/apps/demultiplex/sample_sheet/index.py` and I use `sequence` there

Reply:
I agree with @seallard that the arguments should be more specifik (ie index or sequence)